docs/maintenance.md

```diff
@@ -0,0 +1,49 @@
+# Maintenance mode of storage nodes
+
+## Overview
+
+Storage nodes go into maintenance mode (MM) while internal procedures are required
+to troubleshoot storage service issues. Data on MM nodes MAY be temporarily
+unavailable. Otherwise, such nodes do not differ from fully functional ones.
+Nodes go into MM on their own.
```

Comment:
What did you mean here?

Reply:
could please clarify some details:
- what does "Nodes go into MM on their own" mean?
- "Data on MM nodes MAY be temporarily unavailable" - if Data MAY be unable - what exactly affects on data availability?
- If user issues  "--status maintenance" command, is there any timeout to allow node return back to usual state?

Reply:
> Nodes go into MM on their own.

I mean MM can only be enabled by the node and no one else. I'll try to rephrase.

> what exactly affects on data availability?

The exact procedures performed on node side. Updated this place, pls check it out.

> is there any timeout to allow node return back to usual state?

Do you mean is there a timeout waiting for the state change?

Reply:
I mean the timeout when the node will automatically back to online state.

Reply:
Currently no, the node must return to online state on its own.